src/pytorch_lightning/strategies/launchers/xla_spawn.py

```diff
@@ -51,8 +51,7 @@ class _XLASpawnLauncher(_SpawnLauncher):
     """
 
     def __init__(self, strategy: "Strategy") -> None:
-        super().__init__(strategy)
-        self._start_method = "fork"
```

Comment:
TIL XLASpawn is actually XLAFork. How confusing!

Reply:
Yes exactly :)) The terminology was a bit fuzzy until recently, at least in the torch world. If we look at the `mp.spawn` function here, it also used to support two options spawn and fork despite being grouped under the name "spawn": https://pytorch.org/docs/stable/multiprocessing.html#torch.multiprocessing.spawn